impl/src/main/java/org/jboss/forge/roaster/model/impl/JavaInterfaceMethodImpl.java

```diff
@@ -0,0 +1,187 @@
+package org.jboss.forge.roaster.model.impl;
+
+import java.lang.reflect.Method;
+
+import org.jboss.forge.roaster.model.source.JavaInterfaceSource;
+import org.jboss.forge.roaster.model.source.MethodSource;
+import org.jboss.forge.roaster.model.util.Methods;
+
+public class JavaInterfaceMethodImpl extends MethodImpl<JavaInterfaceSource>
+{
+
+   public JavaInterfaceMethodImpl(JavaInterfaceSource parent)
+   {
+      super(parent);
+   }
+
+   public JavaInterfaceMethodImpl(JavaInterfaceSource parent, Method reflectMethod)
+   {
```

Comment:
Does this constructor do anything different from the parent class? Can we just call super(parent,reflectMethod) instead?